sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -118,8 +173,10 @@ namespace Azure { namespace Core { namespace Http {
   }} // namespace Policies::_internal
 
   /**
-   * @brief HTTP request.
+   * @brief A request message from a client to a server.
    *
+   * @details Includes, within the first line of the message, the HttpMethod to be applied to the
```

Comment:
```suggestion
   * @details Includes, within the first line of the message, the HTTP method to be applied to the
```

Reply:
TBH, I don't completely understand this being applied to a class description "Includes, within the first line of the message"?

Reply:
`HttpMethod` is resolved by doxygen as a type and adds a link to the type.

Reply:
It is going to be self-reference which we decided to avoid

Reply:
It is fine, as user reads the docs, there will be plenty of references to the HttpMethod class in the function signatures, no need to turn everything into a hyperlink.

Reply:
And I still don't know what the comment is trying to say, be it with the link to the class or not :)